HEN_HOUSE/egs++/sources/egs_radionuclide_source/egs_radionuclide_source.h

```diff
@@ -137,20 +137,19 @@ on the number of disintegration events (tracked by <b> \c ishower </b>). This
 is distinct from the <b> \c ncase </b> input parameter, which is the
 number of particles returned by the source (includes relaxations etc.).
 
-A radionuclide source is defined using the following input. Notice that the
-format is similar to \ref EGS_IsotropicSource or \ref EGS_CollimatedSource.
-Be <b>careful using a collimated source</b> - it may neglect effects from the physical
-source on the spectrum, and neglect potential contributions from surrounding
-structures. Restricting the emission angles by collimation generally constitutes
-a physical approximation of your geometry and is an approximate efficiency
-enhancement technique. It is also important to note that the collimated source
-determines the fluence with \f$N/d^2\f$, where \c N is the number of
-disintegrations sampled and \c d is the user-defined minimum distance between the
-source and target shapes.
+A radionuclide source is defined using the following input. It imports a base
+source and uses the base source getNextParticle() invocation to determine decay
+location. This implementation leads to increased random number sampling than
```

Comment:
Why don't we just use an EGS_Shape instead of defining a base source if this is only used to define decay location?

Reply:
Maybe the text is not quite clear - it also uses the direction sampling and getFluence from the base source, so it can uses the more complex source modelling (like a collimated source, which is not just a shape).

Reply:
Ah okay.  Yes, I see further up in the text it says it makes use of getFluence, but maybe it should also mention that it uses the incident directions as well.

Reply:
Done, thanks!